### PR TITLE
feat: create example of automations with python

### PR DESCRIPTION
**Pull request by aballiet**
- Created README.md
- Add other dbt example flow to show reusability

Add automation examples:
- kestra logs db purge for a given namespace
- Python ETL

Currently resource `kestra_namespace_file` does not seem working properly, resource documentation [here](https://registry.terraform.io/providers/kestra-io/kestra/latest/docs/resources/namespace_file)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,77 @@
 # kestra-flows-template
```
Comment (Ben8t) [inline README.md:1]: What about ?

```suggestion
# Kestra Flow Factory with Terraform
```

File: README.md [hunk 2]
```diff
@@ -1 +1,77 @@
...
+
+To develop and deploy Kestra [flows](https://kestra.io/docs/workflow-components/flow), we use [Terraform](https://kestra.io/docs/terraform/guides/configurations).
```
Comment (Ben8t) [inline README.md:3]:
```suggestion
To develop and deploy Kestra [flows](https://kestra.io/docs/workflow-components/flow), Kestra developers can use [Terraform](https://kestra.io/docs/terraform/guides/configurations).

Doing so comes with great advantages:

- Scale the codebase by using the [DRY best practices](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)
- Manage multiple environments thanks to Terraform
- Make reproducibility at the core of your Kestra development
```

File: README.md [hunk 3]
```diff
@@ -1 +1,77 @@
...
+  - `triggers` are meant to be reusable to DRY our code. They should not be declared for a specific flow (except for [flow trigger](https://kestra.io/docs/workflow-components/triggers/flow-trigger))
+
+## Environments
+
+`development` & `production` are deployed on same Kestra instance.
+
+`development` allows you to test your flow before going in production. Make sure you are not working at the same time on it.
+
+`production` requires you to create a PR to validate your changes before being able to apply.
+
```
Comment (Ben8t) [inline README.md:41]:
```suggestion
> Note: Using namespaces to seperate environment within the same Kestra instance is not recommanded for production. You can read more about best practices for Kestra environment and deployment management [here](https://kestra.io/docs/best-practices/from-dev-to-prod)
```

File: README.md [hunk 4]
```diff
@@ -1 +1,77 @@
...
+    - Go on `main` branch
+    - Run `terraform apply` to apply your changes
+
+## [CoreOps] - Module & subflows
+
+In order to provide modular dev experience, we leverage Terraform [modules](https://developer.hashicorp.com/terraform/language/modules/develop) and [subflow](https://kestra.io/docs/workflow-components/subflows) pattern.
+
+- `modules` are used as an abstraction layer for regular user to write workflows (i.e. triggering an Airbyte sync) without having to worry about Kestra syntax, authentication or connection details.
+
+- `subflows` are used for generic tasks where logging is not necessary to have in flow execution.
```
Comment (Ben8t) [inline README.md:70]: > "where logging is not necessary to have in flow execution"

What do you mean by "where logging is not necessary" 🤔 ?

File: modules/purge_kestra/main.tf [hunk 5]
```diff
@@ -11,7 +11,6 @@ module "kestra_purge_db" {
   postgres_database        = "kestra"
   postgres_password_secret = var.postgres_password_secret
   postgres_host            = var.postgres_host
-  show_result_in_logs      = false
```
Comment (Ben8t) [inline modules/purge_kestra/main.tf:14]: Just a comment, FYI there will be a Purge.Logs task in upcoming 0.18 version 😉 probably update here when release is shipped

File: README.md (hunk 3 above)
Comment (aballiet) [reply README.md:47]: Yes great warning !

**Comment by anna-geller** [comment]
@Ben8t @aballiet can this be merged?

**Comment by Ben8t** [comment]
@aballiet do not hesitate to hit "resolve conversation" if done or need to bypass 👍

File: README.md (hunk 4 above)
Comment (aballiet) [reply README.md]: Clarified

**Review by Ben8t — APPROVED** [review]
LGMT ❤️

**Pull request merged**